crates/revm/src/db/states/changes.rs

```diff
@@ -15,9 +15,11 @@ pub struct StateChangeset {
     pub contracts: Vec<(B256, Bytecode)>,
 }
 
+// mattsse: a lot of the type aliases are very hard to read, this looks like Address with some bool marker and a list of storage slots with values, but it is not obvious, Personally I'd prefer structs for these tuples or at the very least docs what these types represent.
```

Comment:
Done in this commit: https://github.com/bluealloy/revm/pull/681/files#diff-4695ed30467f83f567379595f36e895b87a25ad850f296c8c40b3528d72e3af1R21

And renamed it for consitency